index.html

```diff
@@ -0,0 +1,135 @@
+                        </ul>
+                    </div>
+                </div>
+            <div class="introContent">
+        <div class="content">
+            <h1>"Hello! Music Lovers"</h1>
+            <h2>Rolling Stone 2023: Uniting Music, Joy, and Memories!"</h2>
+            <p>Welcome to Rolling Stone 2023, where the magic of music comes to life! Get ready for an unforgettable experience as we bring together the world's most talented artists, the thrill of live performances, and the vibrant spirit of a music festival that will leave you exhilarated.</p>
+            <h3>August 25 ~ August 27</h3>
+            <h4>@The Garden of Five Senses,Delhi</h4>
+        </div>
+        </div>
+        </Section>
+        <!-- Program -->
+        <section class="program">
+            <h3>Main Program</h3>
+            <div class="underLine"></div>
+            <div class="programContainer">
+            <div class="programCards">
+                <article class="programEvents">
+                    <img src="./resources/MainProgramIcons/1.png" alt="programImg">
+                    <h4>Silent Disco</h4>
+                    <p>Experience the latest trend in entertainment with our silent disco nights.</p>
+                </article>
+                <article class="programEvents">
+                    <img src="./resources/MainProgramIcons/2.png" alt="programImg">
+                    <h4>Food Frenzy</h4>
+                    <p>Stay refreshed throughout the concert with our beverage collaboration.</p>
+                </article>
+                <article class="programEvents">
+                    <img src="./resources/MainProgramIcons/3.png" alt="programImg">
+                    <h4>Drinks</h4>
+                    <p>Indulge in a culinary adventure with a wide array of gourmet food stalls.</p>
+                </article>
+                <article class="programEvents">
+                    <img src="./resources/MainProgramIcons/4.png" alt="programImg">
+                    <h4>Art & Music</h4>
+                    <p> Immerse yourself in a world of creativity with interactive art exhibitions.</p>
+                </article>
+                <article class="programEvents">
+                    <img src="./resources/MainProgramIcons/5.png" alt="programImg">
+                    <h4>VIP lounge</h4>
+                    <p>Upgrade your concert night with exclusive access to our VIP lounge.</p>
+                </article>
+            </div>
+        </div>
+            <div class="Joining">
+                <button type="button" class="joinBtn">Join Rolling Stone: 2023</button>
+            </div>
+            </section>
+            <!-- Lineup -->
+            <section class="Featured">
+            <h2>Featured Speakers</h2>
+            <div class="underLine"></div>
+            <div id="speakers" class="speak"></div>
+            <button class="moreBtn" id="seeMoreButton">See More</button>
+              </section>
```

Comment:
- [ ] As requested by the previous reviewer, Please ensure that your dropdown icon and texts align correctly and identical in the design to match the design, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustrations of yours and the given Design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Student See more button](https://github.com/DevKaranJ/Capstone-project-Music-Festival-Website/assets/43528594/e663a540-0e69-49b0-b604-c21c5de4a8de)


</td>
      <td>

![design button](https://github.com/DevKaranJ/Capstone-project-Music-Festival-Website/assets/43528594/b9b90a5a-c2ab-4b7d-9e7f-a2895e0d6077)

</td>
    </tr>
  </table>